walkthroughs/week-5-tdd/project/src/test/java/com/google/sps/FindMeetingQueryTest.java

```diff
@@ -49,12 +49,14 @@
   private static final int TIME_0400PM = TimeRange.getTimeInMinutes(16, 0);
   private static final int TIME_0430PM = TimeRange.getTimeInMinutes(16, 30);
  
+  private static final int DURATION_NEGATIVE = -1;
   private static final int DURATION_15_MINUTES = 15;
   private static final int DURATION_30_MINUTES = 30;
   private static final int DURATION_60_MINUTES = 60;
   private static final int DURATION_90_MINUTES = 90;
   private static final int DURATION_1_HOUR = 60;
   private static final int DURATION_2_HOUR = 120;
+  private static final int DURATION_WHOLE_DAY = 1440;
```

Comment:
Usually when writing numbers, prefer to write as `24 * 60` instead of `1440`, which explains where the 1440 comes from. Another way is to add a comment for 1440.